src/js/controller/controller.js

```diff
@@ -376,6 +376,10 @@ Object.assign(Controller.prototype, {
 
             const playReason = _getReason(meta);
             _model.set('playReason', playReason);
+            // Stop autoplay behavior if the video is started by the user or an api call
+            if (playReason === 'interaction' || playReason === 'external') {
+                _model.set('playOnViewable', false);
+            }
```

Comment:
This is fine for now, but we need to make a tech-debt note that model properties shouldn't span view-controls code and player-model-controller. Everything "autostart" and "playOnViewable" is split up between controls and controller instead of being encapsulated in one place, like a module that listens to model changes ("change:playReason" where reason === "interaction").